src/esr/mod.rs

```diff
@@ -161,6 +161,16 @@ pub fn decode(esr: u64) -> Result<Vec<FieldInfo>, DecodeError> {
             decode_iss_fp(iss.value)?,
             None,
         ),
+        0b010110 => (
+            "HVC instruction execution in AArch64 state",
+            decode_iss_hvc(iss.value)?,
+            None,
+        ),
+        0b010111 => (
+            "SMC instruction execution in AArch64 state",
+            decode_iss_hvc(iss.value)?,
+            None,
+        ),
```

Comment:
Let's keep these sorted in numerical order.

Reply:
Of course, this is done!